bindings/go/main.go

```diff
@@ -49,14 +49,14 @@ var (
 // will return nil.
 func makeErrorFromRet(ret C.C_KZG_RET) error {
 	switch ret {
+	case C.C_KZG_OK:
```

Comment:
This is not expected to happen, all current callsites avoid invoking it for this path

Reply:
I agree. Just wanted to maintain the same order as C. I would actually like to remove this case & state that it should only be called if `ret` isn't `C_KZG_RET`. How would you feel about this?

Reply:
Yeah fine by me as long as it doesn't panix